app/code/core/Mage/Adminhtml/Block/Store/Switcher.php

```diff
@@ -223,38 +223,4 @@ public function hasDefaultOption($hasDefaultOption = null)
         }
         return $this->_hasDefaultOption;
     }
-
-    /**
-     * Return url for store switcher hint
-     *
-     * @return string
-     */
-    public function getHintUrl()
-    {
-        if (null === $this->_hintUrl) {
-            $this->_hintUrl = Mage::helper('core/hint')->getHintByCode(self::XPATH_HINT_KEY);
```

Comment:
I think, that we should also remove constant `XPATH_HINT_KEY`, which is currently used nowhere

Reply:
you're totally right, I have a problem tho, I've deleted that forked repo cause I messed it up and I think I can't modify the PR anymore :-|

Reply:
its a bit "complicated", but you can checkout a PullRequest directly, see step5: https://docs.github.com/en/github/collaborating-with-issues-and-pull-requests/checking-out-pull-requests-locally

Reply:
I tried but I think I would be able to do actually update the PR only if I had write access to this repo. If seems (https://docs.github.com/en/github/collaborating-with-issues-and-pull-requests/checking-out-pull-requests-locally) that I can checkout a PR but then I can only create another one... so if you agree I'd merge this one and I'll create another PR later, just to remove the constant, if that's ok.

Reply:
@fballiano I think, that you should create new PR now (with complete code) and link this pull request inside. This PR should be closed, if is not possible to edit it.

Reply:
it seems a bit overkill just to remove an unused constant... anyways... I'll do that, let's not get stuck on that.